src/fdk_rdf_parser/organizations/organizations_client.py

```diff
@@ -0,0 +1,19 @@
+from typing import Optional
+from urllib import error, request
+
+from .utils import organizationUrl
+
+
+def getRdfOrgData(orgnr: Optional[str]) -> Optional[str]:
+
+    req = request.Request(
+        url=organizationUrl(orgnr), headers={"Accept": "text/turtle"}, method="GET",
+    )
+
+    try:
+        with request.urlopen(req) as response:
```

Comment:
I'd strongly recommend using Python's [requests](https://requests.readthedocs.io/en/master/) package instead of urllib.

Reply:
618ec97